pkg/provider/github/client.go

```diff
@@ -145,8 +145,7 @@ func (g *Client) GetSecretMap(ctx context.Context, ref esv1.ExternalSecretDataRe
 	return nil, fmt.Errorf("not implemented - this provider supports write-only operations")
 }
 
-func (g *Client) Close(ctx context.Context) error {
-	ctx.Done()
+func (g *Client) Close(_ context.Context) error {
```

Comment:
did you leave this change in by accident?

Reply:
I can remove it, but I thought we talked about it https://github.com/external-secrets/external-secrets/pull/5160#discussion_r2286961075